client/ayon_houdini/plugins/publish/extract_last_published.py

```diff
@@ -0,0 +1,99 @@
+import os
+import shutil
+
+import clique
+import pyblish.api
+
+from ayon_core.lib import collect_frames
+from ayon_houdini.api import plugin
+
+
+class ExtractLastPublished(plugin.HoudiniExtractorPlugin):
+    """
+    Generic Extractor that copies files from last published
+    to staging directory.
+    It works only if instance data includes "last_version_published_files"
+    and there are frames to fix.
+
+    The files from last published are base of files which will be extended/fixed for specific
+    frames.
+    """
```

Comment:
I don't get it. If this is _generic_? How come `ayon-core` does not have this yet... but it does have `CollectFrameToFix` plug-in? Shouldn't there be some other implementation of this then somewhere that does _exactly_ this?

EDIT: AH, it's from Nuke [here](https://github.com/ynput/ayon-nuke/blob/43726e8ff8ca7a57f9169995b03447a413af8658/client/ayon_nuke/plugins/publish/extract_render_local.py#L70-L218).

This should come with a massive `# TODO` block screaming that this is taken 1:1 from `ayon-nuke` and should transition to `ayon-core`. Or we transition that now I'd say.

Reply:
> transition that now I'd say.

I've no problem doing that but I'm afraid we fell in the same situation as https://github.com/ynput/ayon-core/pull/839#issuecomment-2293777570